src/core/lib/gprpp/fork.cc

```diff
@@ -228,5 +235,6 @@ void Fork::AwaitThreads() {
 
 std::atomic<bool> Fork::support_enabled_(false);
 bool Fork::override_enabled_ = false;
-Fork::child_postfork_func Fork::reset_child_polling_engine_ = nullptr;
+std::vector<Fork::child_postfork_func>* Fork::reset_child_polling_engine_ =
```

Comment:
I don't think this needs to be a pointer, the nullptr check doesn't seem to have much purpose other than to lazy initialize the vector.

Reply:
Is `std::vector<T*>` trivially destructible? I know this is `static`, not technically global, but I figured the same rules applied.

Reply:
I didn't realize this was a global, you're right. You could use the `NoDestruct` types src/core/lib/gprpp/no_destruct.h, but this works as well.